<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wifi</class>
 <widget class="QWidget" name="Wifi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="ui_adapterLabel">
     <property name="text">
      <string>Wireless adapter detected! 00:00:00:00:00:00</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Available Networks</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="ui_networkList">
        <item>
         <property name="text">
          <string>Wguest</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nease</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Thompson</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
       <property name="horizontalSpacing">
        <number>2</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QLabel" name="ui_ipLabel">
         <property name="text">
          <string>192.168.1.108</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>IP:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>SSID:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="ui_ssidLabel">
         <property name="text">
          <string>Thompson</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="ui_refreshButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>QPushButton{
border:0px;
background-image:url(:/actions/rivet80x30L.png);
color:black;
}
QPushButton:pressed{
border:0px;
background-image:url(:/actions/rivet80x30D.png);
color:white;
}</string>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ui_connectButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>QPushButton{
border:0px;
background-image:url(:/actions/rivet80x30L.png);
color:black;
}
QPushButton:pressed{
border:0px;
background-image:url(:/actions/rivet80x30D.png);
color:white;
}</string>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
